arch/xtensa/src/esp32/esp32_wireless.c

```diff
@@ -351,6 +351,85 @@ static void esp_wifi_set_log_level(void)
  * Public Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: esp_wifi_to_errno
+ *
+ * Description:
+ *   Transform from ESP Wi-Fi error code to NuttX error code
+ *
+ * Input Parameters:
+ *   err - ESP Wi-Fi error code
+ *
+ * Returned Value:
+ *   NuttX error code defined in errno.h
+ *
+ ****************************************************************************/
+
+int32_t esp_wifi_to_errno(int err)
```

Comment:
Why errno return value is `int32_t` and not `int`?

Reply:
No specific reason. Similar functions for the other architectures use the same type (for RISC-V, for instance: https://github.com/apache/incubator-nuttx/blob/2c4963dcd6f9785d6d3e5b327e22fd995cbc81a7/arch/risc-v/src/common/espressif/esp_wifi_utils.h#L112). We can change it for `int` for all of them, I just kept the same for now.

Reply:
Ok